Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class FileCleaner
+    {
+        private const char DollarSign = '£';
+
+        /// <summary>
+        /// Cleans up dirtyFileName 
+        /// </summary>
+        /// <param name="dirtyFile">Dirty file with "_" placed in random places.</param>
+        /// <param name="cleanedFile">Cleaned up file without any "_".</param>
+        public static void Clean(string dirtyFile, string cleanedFile)
+        {
+            if (!isValidFile(dirtyFile, cleanedFile))
+            {
+                return;
+            }
+
+            var corruptedBalanceLines = File.ReadAllLines(dirtyFile);
+            var fixedBalanceLines = new string[corruptedBalanceLines.Length];
+
+            File.WriteAllText(cleanedFile, FixCorruptedBalanceFile(corruptedBalanceLines, fixedBalanceLines));
+
+            DisplayStasticalData(CreateBalanceLinesForTextTable(fixedBalanceLines));
+        }
+
+        private static bool isValidFile(string dirtyFile, string cleanedFile)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'isValidFile' to match pascal case naming rules, consider using 'IsValidFile'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)